docs/stories/13 - Cold Hands, Stone Heart.adoc

```diff
@@ -0,0 +1,135 @@
+a spark that had danced its way out of the firepit. The cloth had smoked, then caught a small flame, and the sparks had awakened me.
+
+This is dangerous, I know. Fire would destroy this home if it could. I am certain that destruction is the nature of fire. I push the cloth into the firepit, and smother the burnt spot on the floor. Still, there was much here that could burn. Rin's blanket. Mother's broom. Father's coat. I did not want this place to be destroyed.
+
+There is no need to waken them though. They've taken care of me, and now I wish to do the same for them. Before I go to sleep, I put the fire out.
+
+'''
+
+When I awaken it is cold. Not the chill of yesterday's morning, but like that first night outside. My mouth is rough and dry. The pain in my head is different than when I woke up then, but in many ways it is now much worse. And there is Rin, and there are Father and Mother, their faces white, their lips blue.
+
+Oh no.
+
+Rin's skin is cold, and no breath comes from her. I rebuild the fire but she stays cold, unmoving, far past sleep. They all are.
+
+Oh Mother, what have I done?
+
+Oh Father, better that you had killed me.
+
+I wish I had never come to this place. I wish I had never done any of this. Had I never come here, their lives would have continued undisturbed. Had I never hunted the boar, we'd never have had such a celebration. Had I never cared for their sleeping forms they'd not have frozen to death.
+
+And for the first time, my voice calls forth. I shout, I scream, I cry out and my voice flies to the heavens I once knew and to the darkness down below. It carries across this world and to other worlds beyond. It calls to Father and Mother. It calls to Rin. I call to them and plead, I'm sorry! I call to them and beg, Come back!
+
+Please Mother, I whisper, Let them find their way back.
+
+There is nothing left for me. I pack up what I can in an old sack they must have used when they had to make long journeys. I take the porridge but I leave the boar meat and the wine. I do not think I'll have either again.
+
+I head south east, back through the copse of trees. Back to the oogashi trees where Rin found me. To the Oogashi. Close enough.
+
+Past them, amid the mume , I find another frozen blossom. In the daylight it shimmers in the crystalline ice more vibrantly than in starlight. The sun shines through the translucent petals, breaking into a thousand points of light in the sharp frost. Its beauty makes my heart ache. Father, Mother, Rin, all gone. It hurts, I wish nothing more than to be able to look upon this frozen flower, to have one thing of beauty, one moment of quiet, and I reach for it-
+
+-and I stop. I will not shatter another blossom with my thoughtless actions.
+
+South east.
+
+image::story_images/13_cold_hands_stone_heart.png[sword,200,200,align="center"]
```

Comment:
Please add a `{empty} +` between image and text, with a blank line before and after it.